operator/pkg/tasks/deint/component.go

```diff
@@ -0,0 +1,95 @@
+package tasks
+
+import (
+	"errors"
+	"fmt"
+
+	"k8s.io/klog/v2"
+
+	"github.com/karmada-io/karmada/operator/pkg/constants"
+	"github.com/karmada-io/karmada/operator/pkg/util"
+	"github.com/karmada-io/karmada/operator/pkg/util/apiclient"
+	"github.com/karmada-io/karmada/operator/pkg/workflow"
+)
+
+// NewRemoveComponentTask init a remove karmada components task
+func NewRemoveComponentTask() workflow.Task {
+	return workflow.Task{
+		Name:        "remove-component",
+		Run:         runRemoveComponent,
+		RunSubTasks: true,
+		Tasks: []workflow.Task{
+			newRemoveComponentSubTask(constants.KarmadaWebhookComponent, util.KarmadaWebhookName),
```

Comment:
I remember there is some svc installed. Do we need to uninstall them when deinit?

Reply:
I want to set the service OwnReference to their workflow. so, we only delete the worklfow.
@Poor12  How do you think?

Reply:
For now I think it's okay. I wonder if there is a scenario where we need to consider deployment and svc separately.

Reply:
Ok, If the service resource exists alone, it doesn't to make sense. I seem not encounter the scenario.